public/app/features/admin/state/actions.ts

```diff
@@ -30,6 +41,9 @@ export const userSyncFailedAction = actionCreatorFactory('USER_SYNC_FAILED').cre
 export const revokeUserSessionAction = actionCreatorFactory('REVOKE_USER_SESSION').create();
 export const revokeAllUserSessionsAction = actionCreatorFactory('REVOKE_ALL_USER_SESSIONS').create();
 
+export const userProfileLoadedAction = actionCreatorFactory<UserDTO>('admin/user/PROFILE_LOADED').create();
```

Comment:
Nitpick: best practices is to introduce an interface for your payloads. (https://github.com/grafana/grafana/blob/master/contribute/style-guides/redux.md#actioncreatorfactory)
```suggestion
export interface UserProfileLoaded {
 user: UserDTO;
}
export const userProfileLoadedAction = actionCreatorFactory<UserDTO>('admin/user/PROFILE_LOADED').create();
```